pkg/scheduler/extender_test.go

```diff
@@ -424,15 +390,15 @@ func TestConvertToMetaVictims(t *testing.T) {
 			nodeNameToVictims: map[string]*extenderv1.Victims{
 				"node1": {
 					Pods: []*v1.Pod{
-						{ObjectMeta: metav1.ObjectMeta{Name: "pod1", UID: "uid1"}},
-						{ObjectMeta: metav1.ObjectMeta{Name: "pod3", UID: "uid3"}},
+						st.MakePod().Name("pod1").UID("uid1").Obj(),
```

Comment:
we can make MakePod() accept variadic parameters so as to avoid significant changes.

> it seems that in many places we create pods without names

yup, UTs don't really need a pod name.

Reply:
> we can make MakePod() accept variadic parameters so as to avoid significant changes.
What would the `MakePod()` look like for this case?

Are you imagining something like?

```go
func MakePod(name ...string) *PodWrapper {
	wrapper := &PodWrapper{v1.Pod{}}
	if len(name) >= 1 {
                wrapper.SetName(name[0])
        }
        return wrapper
}
```

This will be kind of weird if the caller is specifying multiple values as only the first will be used.

> yup, UTs don't really need a pod name.
I agree here that UTs don't necessarily need specify a name so enforcing may not be ideal.

Maybe we can consider doing a separate clean up for this case in a different PR?

Reply:
I'm fine with leaving it as a follow-up.

Reply:
sgtm